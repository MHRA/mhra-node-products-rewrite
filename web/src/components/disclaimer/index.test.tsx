import renderer from 'react-test-renderer';
import React from 'react';
import Disclaimer from './index';

describe(Disclaimer, () => {
  it('should render', () => {
    const component = renderer
      .create(
        <Disclaimer onDisclaimerAgree={jest.fn()} searchTerm={'Perinola'} />,
      )
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
