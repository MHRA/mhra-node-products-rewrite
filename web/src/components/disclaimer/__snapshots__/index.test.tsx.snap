// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Disclaimer should render 1`] = `
<section
  className="sc-hKMtZM cracHi"
>
  <p>
    Please read the following information and tick the box to proceed to view the product information in pdf format.
  </p>
  <div
    className="sc-dkzDqf dyqvZI"
  >
    <h3>
      Disclaimer:
    </h3>
    <p>
      I understand that this information is a copy of the Summary of Product Characteristics and patient information leaflet for a medicine, which outline the conditions under which the medicine should be used and information on its known safety.
    </p>
    <p>
      I understand that this information may be updated several times during the productâ€™s lifecycle, and that there could be differences between the information shown here and other information in the public domain.
    </p>
    <p>
      I understand that the MHRA is unable to offer medical advice and that if a patient has any questions about a medicine they are taking they should contact their doctor or pharmacist. Patients should not stop taking any prescribed medicines without first speaking to a healthcare professional. Suspected adverse reactions to a medicine can be reported to us on a
       
      <a
        href="https://yellowcard.mhra.gov.uk/"
        rel="noopener noreferrer"
        target="_blank"
      >
        Yellow Card.
      </a>
    </p>
    <p>
      I understand that the MHRA has used its best endeavours in publishing this information, but accept that the information may not be the most up to date version for this product.
    </p>
    <p>
      To view details for 
      <em>
        Perinola
      </em>
      , please read and accept the disclaimer.
    </p>
    <form>
      <div>
        <input
          className="sc-bczRLJ ldzjBD"
          id="agree-checkbox"
          name="agree"
          onChange={[Function]}
          type="checkbox"
        />
        <label
          htmlFor="agree-checkbox"
        >
          I have read and understand the disclaimer.
        </label>
      </div>
      <button
        disabled={true}
        onClick={[MockFunction]}
        type="submit"
      >
        Agree
      </button>
    </form>
  </div>
</section>
`;
